### DISCUSSION TITLE
posts/company-departure-after-3-years/

### DISCUSSION THREAD

**Discussion by giscus** [WincerChan/blog-comments#3] [Announcements]
# posts/company-departure-after-3-years/

我曾在离开上家公司时 做过总结 ，离职的原因肯定是公司有些毛病你无法忍受却又无力改变，如今看来，这句话多少有些片面：因为我所在的部门同事和领导关系融洽、工作氛围还算轻松，可我依然决定了离开。这倒不是说公司没有毛病，只是说毛病还没有到让我无法忍受的地步，相比于有同事打小报告、工作氛围卷得要死的前公司来说，简直是小巫见大巫。

https://blog.itswincer.com/posts/company-departure-after-3-years/

**Comment by yzyyz1387** [comment]
还没参与工作，无法切身体会，但能感受字里行间的无奈啊，越读感觉事情越严重，确实之前经常来看博客发现你一直没写点什么，哈哈，现在缓过来了吗

**Reply by WincerChan** [reply]
现在休息了三个月了，倒是缓过来了不少。哎，如果每段工作经历都需要空闲几个月时间才能缓过来，那工作得多无奈啊…
